src/crystal/system/unix/time.cr

```diff
@@ -49,9 +49,10 @@ module Crystal::System::Time
 
   def self.monotonic
     {% if flag?(:darwin) %}
-      info = mach_timebase_info
-      nanoseconds = LibC.mach_absolute_time.to_i64 * info.numer / info.denom
-      {nanoseconds / 1_000_000_000, nanoseconds.remainder(1_000_000_000).to_i32}
+      absolute_time = LibC.mach_absolute_time * mach_ticks_per_nanosecond
```

Comment:
Please revert this last change. It's really confusing because usually you'd divide by "ticks per nanosecond" but this is actually "nanoseconds per tick".